package-lock.json

```diff
@@ -33,7 +33,7 @@
         "csstype": "^3.0.9",
         "darkreader": "^4.9.39",
         "dbus-next": "^0.10.2",
-        "debug": "4.3.4",
+        "debug": "^4.3.2",
```

Comment:
can we please use `4.3.3` as the specific version if that's what you want to use? I don't prefer open-ended version definitions - since that can then cause weird errors across machines.